examples/shadowmap/main.js

```diff
@@ -1133,15 +1133,26 @@
                 15 + 35 * num
             ];
 
-            var shadowSettings = new osgShadow.ShadowSettings( this._config );
 
-            var mapres = parseInt( this._config[ 'textureSize' ] );
-            shadowSettings.setTextureSize( mapres );
+            var shadowSettings;
+            if ( this._config[ 'atlas' ] ) {
+
+                shadowSettings = this._shadowSettings[ num ];
+                shadowMap = this._shadowMapAtlas.addLight( light, shadowSettings );
```

Comment:
It does not needs shadowSettings, can be set once at the creation of ShadowAtlas